AmethystRuntime/src/AmethystRuntime.cpp

```diff
@@ -53,6 +53,8 @@ void AmethystRuntime::LoadModDlls()
         mLoadedMods.emplace_back(modName);
     }
 
+    this->mAmethystContext.mMods = mLoadedMods;
```

Comment:
Sorry I missed this. mLoadedMods should be removed, we do not want a copy of it in mMods, we want to directly use the one in mMods

Reply:
done...